pandas/plotting/_core.py

```diff
@@ -1723,7 +1724,8 @@ def result(self):
 _all_kinds = _common_kinds + _dataframe_kinds + _series_kinds
 
 _klasses = [LinePlot, BarPlot, BarhPlot, KdePlot, HistPlot, BoxPlot,
-            ScatterPlot, HexBinPlot, AreaPlot, PiePlot]
+            ScatterPlot, HexBinPlot, AreaPlot, PiePlot] \
+    # type: List[Type[MPLPlot]]
```

Comment:
Should just be `List[MPLPlot]` I think - does that give you some kind of error?

Reply:
Yes, `_klasses` contains the classes/types themselves, not instances of those types, so using `List[MPLPlot]` errors out with lines like `error: List item 0 has incompatible type "Type[LinePlot]"; expected "MPLPlot"`.